torch/_tensor.py

```diff
@@ -878,8 +878,7 @@ def __format__(self, format_spec):
 
     @_handle_torch_function_and_wrap_type_error_to_not_implemented
     def __rpow__(self, other):
-        dtype = torch.result_type(other, self)
-        return torch.tensor(other, dtype=dtype, device=self.device) ** self
+        return torch.pow(other, self)
```

Comment:
Why didn't we do this before?

And is __rpow__ just ** ?

Reply:
I can guess:
- The scalar overload didn't exist
- We didn't care about tensor() before tracing/compilation happened.

rpow is `reversed-**`